avocado/plugins/variants.py

```diff
@@ -67,7 +67,7 @@ def configure(self, parser):
                             "Shows the node content (variables)")
         env_parser = parser.add_argument_group("environment view options")
         env_parser.add_argument('-d', '--debug', action='store_true',
-                                dest="mux_debug", default=False,
+                                dest="debug", default=False,
```

Comment:
This debug is really specific option. I think `variant_debug` or `varianter_debug` would be more appropriate to prevent possible clash of this argument with general purpose `--debug` argument.

Reply:
I thought about that as well, but this is limited to the `variants` command so it seems OK to not need a prefix.  All other command line options are *not* using a prefix, so only this one using it would actually look odd.  What do you think?

Reply:
Not really, if you introduce subparser "debug", you make it impossible for main parser to define it as well. And it's even worse because the "debug" param is used in standard execution as well so if theoretically someone introduced "run" plugin with "debug" argument, it'd also start using TreeNodeDebug instead of TreeNode...

Reply:
Ok, I see @ldoktor point here. While harmless currently, it can lead us to future issues, so let's just use a less break-prone naming. Please have a look at https://github.com/avocado-framework/avocado/pull/2360